lib/cog/repository/bundles.ex

```diff
@@ -39,6 +39,8 @@ defmodule Cog.Repository.Bundles do
   @install_types [:normal, :force]
   @default_install_type :normal
 
+  @minimum_bundle_config_version Spanner.Config.minimum_config_version
```

Comment:
I see.

I still think `set_bundle_version_status` could use some explanatory comments to aid understanding.